src/components/aktivitetskrav/vurdering/vurderAktivitetskravSkjemaTypes.ts

```diff
@@ -1,4 +1,30 @@
+import {
+  AvventVurderingArsak,
+  OppfyltVurderingArsak,
+  UnntakVurderingArsak,
+} from "@/data/aktivitetskrav/aktivitetskravTypes";
+
 export interface VurderAktivitetskravSkjemaProps {
   setModalOpen: (isOpen: boolean) => void;
   aktivitetskravUuid: string | undefined;
 }
+
+export interface AktivitetskravSkjemaValues {
+  beskrivelse: string;
+}
+
+export interface AvventAktivitetskravSkjemaValues
+  extends AktivitetskravSkjemaValues {
+  arsaker: AvventVurderingArsak[];
+  fristDato?: string;
+}
+
+export interface OppfyltAktivitetskravSkjemaValues
+  extends AktivitetskravSkjemaValues {
+  arsak: OppfyltVurderingArsak;
+}
+
+export interface UnntakAktivitetskravSkjemaValues
+  extends AktivitetskravSkjemaValues {
+  arsak: UnntakVurderingArsak;
+}
```

Comment:
Synes personlig at slike interfaces lever bedre sammen der de brukes 😊 Man får jo minimalt med kodeduplisering ved å drive med arv, men synes i de fleste tilfeller at det gjør koden vanskeligere å forstå

Reply:
Ja, tror jeg tenkte at de skulle brukes flere steder dersom det ble egne komponenter for hvert skjema-felt skilt ut i egne filter. Nå ble det ikke så mye av det så da kan sikkert noen av disse flyttes til skjema-komponenten. Tar en rydding av det i neste PR.